client/src/main/java/io/pravega/client/stream/EventStreamReader.java

```diff
@@ -33,14 +33,17 @@
     
     /**
      * Gets the next event in the stream. If there are no events currently available this will block up for
-     * timeout waiting for them to arrive. If none do, an EventRead will be returned with null for
+     * timeoutMillis waiting for them to arrive. If none do, an EventRead will be returned with null for
      * {@link EventRead#getEvent()}. (As well as for most other fields)
      *
      * An EventRead with null for {@link EventRead#getEvent()} is returned when the Reader has read all events up to the
      * configured end {@link StreamCut} specified using {@link ReaderGroupConfig}.
+     *<p>
```

Comment:
I'm not sure whether we normally use this kind of markup or not in our javadoc. If not, we may need to remove this (and perhaps other occurrences in the code).

Reply:
Used the <p>paragraph tag is used to align the java doc. ref: https://www.oracle.com/technetwork/articles/java/index-137868.html